README.md

```diff
@@ -97,6 +97,22 @@ The `dynatrace-bootstrapper` is a small CLI binary built into a [Dynatrace CodeM
 - This is an **optional** arg
 - The `--attribute-container` arg defines the passed in Container attributes that will be used to configure the metadata-enrichment and injected CodeModule. It is a JSON formatted string.
 
+#### `--silent`
```

Comment:
i like examples, do you know you can add it as parameter to cobra struct:


```
RunExample             = `
example here :) 
`

	cmd := &cobra.Command{
		Use:                "run",
		Short:              "...",
		Long:               "...",
		RunE:               ....,
		Example:            RunExample,
		DisableFlagParsing: true,
	}
```

Reply:
BTW you can also rename to `--suppress-errors`, silent a bit unclear - but ok with me .

Reply:
> do you know you can add it as parameter to cobra struct

I would just update the `usage` param in `cmd.PersistentFlags().StringVar(...)` to also contain these examples
because what you are showing is the example for the whole command which can be rather large, if you look at the sample 😅

Reply:
=) just shared prev expirience ;) non blocking comment